docs/architecture/05_kafka.md

```
@@ -136,31 +140,27 @@
-    "portoneTransactionId": "imp_123456",
-    "scheduleId": "schedule-uuid-111",
-    "seatIds": ["seat-uuid-001", "seat-uuid-002"]
-  }
+  "paymentKey": "payment-key-abc",
+  "reservationId": "reservation-uuid-321",
+  "amount": 200000.00,
+  "paidAt": "2026-01-20T10:00:00",
+  "scheduleId": "schedule-uuid-111",
+  "seatIds": ["seat-uuid-001", "seat-uuid-002"]
 }
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)